frontend/src/components/comments/comment.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import Header from './header';
```

Comment:
it's more explicit to use names like **CommentHeader** instead of just **Header**